include/boost/gil/extension/rasterization/ellipse.hpp

```diff
@@ -164,20 +178,32 @@ struct midpoint_ellipse_rasterizer
     ///        in the function 'draw_curve' for drawing the desired ellipse.
     /// \param view - Gil view of image on which the elliptical curve is to be drawn.
     /// \param pixel - Pixel value for the elliptical curve to be drawn.
-    /// \param center - Point containing positive integer x co-ordinate and y co-ordinate of the
-    /// center respectively.
-    /// \param semi_axes - Point containing positive integer lengths of horizontal semi-axis
-    /// and vertical semi-axis respectively.
     /// \tparam View - Type of input image view.
     /// \tparam Pixel - Type of pixel. Must be compatible to the pixel type of the image view
     template<typename View, typename Pixel>
-    void operator()(View& view, Pixel const& pixel,
-        point<unsigned int> center, point<unsigned int> semi_axes)
+    void operator()(View& view, Pixel const& pixel) const
     {
-        draw_curve(view, pixel, center, obtain_trajectory(semi_axes));
+        draw_curve(view, obtain_trajectory());
```

Comment:
@marco-langer AFAICT, this `draw_curve(view, obtain_trajectory());` invocation is not correct as the `draw_curve` still takes 3 arguments: `view`, `pixel` and `trajectory_points`.

The CI jobs for this PR were green though. It looks like this part is not covered by tests. 
It will fail during compilation of `example/rasterizer_ellipse.cpp` though.

Reply:
I will fix it and also add test cases

Reply:
Thank you!